RELEASE-NOTES.txt

```diff
@@ -1,6 +1,6 @@
 24.4
 -----
-
+* [**] Block editor: Introduce VideoPress v5 support, to fix issues using video block with dotcom and Jetpack sites [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6634]
```

Comment:
Additionally, since VideoPress v5 is a Jetpack block and it won't be available in the WP app, we should consider adding the prefix `[Jetpack-only]`.